### ISSUE TITLE
Price not shown in locale format

### ISSUE THREAD

**Issue by peterjaap** [algolia/algoliasearch-magento-2#615]
**Do you want to request a *feature* or report a *bug*?**
Bug

**Bug: What is the current behavior?**
After saving a product, the Algolia index for the German storeview does not contain the German-locale formatted price (`24,95 €`), but the default storeview formatted price (`€24,95`);

![image](https://user-images.githubusercontent.com/431360/48564232-47dc0300-e8f6-11e8-8016-8343355818e8.png)

**Bug: What is the expected behavior?**
It should contain the German-locale formatted price (with the euro sign behind the price instead of in front of it), like this;

![image](https://user-images.githubusercontent.com/431360/48564274-56c2b580-e8f6-11e8-91f0-4fb740b7d6a4.png)

**Bug: What is the proposed solution?**
Not sure. Since [the code](https://github.com/algolia/algoliasearch-magento-2/blob/master/Helper/Entity/Product/PriceManager.php#L109) does try to fetch it through the priceCurrency `format` function;

```
                $customData[$field][$currencyCode]['default_formated'] = $this->priceCurrency->format(
                    $price,
                    false,
                    PriceCurrencyInterface::DEFAULT_PRECISION,
                    $store,
                    $currencyCode
                );
```

**What is the version of Magento and of Algolia extension you are using? Always use the latest version of the extension one before opening a bug issue.**

Magento 2.2.6, Algolia 1.7.1

**Comment by peterjaap** [comment]
Hang on. Just saw there were updates in the PriceManager since 1.7.1. Closing for now while I upgrade and test again.

**Comment by peterjaap** [comment]
Confirmed fixed in at least 1.8.3;

![image](https://user-images.githubusercontent.com/431360/48565148-23812600-e8f8-11e8-9eb6-7492efe9918e.png)
